command/agent/internal/ctmanager/runner_config.go

```diff
@@ -11,7 +11,6 @@ import (
 	ctconfig "github.com/hashicorp/consul-template/config"
 	ctlogging "github.com/hashicorp/consul-template/logging"
 	"github.com/hashicorp/go-hclog"
-
```

Comment:
Nobody likes the extra line.

Reply:
<img src="https://media3.giphy.com/media/l4FB7E55OQPoi4H6w/giphy.gif"/>